scripts/python/phdf_diff.py

```diff
@@ -64,51 +64,28 @@ def addPath():
     #sys.path.insert(0,myPath+'/../vis/python')
     #sys.path.insert(0,myPath+'/vis/python')
 
-if __name__ == "__main__":
-    addPath()
+def compare(files, all=False, brief=True, quiet=False, one=False, tol=1.0e-12):
+    """ compares two hdf files """
 
     #**************
     # import Reader
     #**************
     from phdf import phdf
 
-
     #**************
     # Reader Help
     #**************
     # for help  on phdf uncomment following line
     # print(help(phdf))
 
-    # process arguments
-    input = processArgs()
-
-    brief=input.b
-    quiet=input.q
-    one = input.o
-
-    # set all only if brief not set
-    if brief or quiet:
-        all=False
-    else:
-        all = input.a
-    files = input.files
-
-    if input.tol is not None:
-        tol = float(input.tol)
-    else:
-        tol = 1.0e-12
-
-
-    if len(files) != 2:
-        Usage()
-        exit(1)
 
     # Load first file and print info
     try:
         f0 = phdf(files[0])
         if not quiet: print(f0)
     except:
-        exit(1)
+        print("whoosp")
```

Comment:
Could we get a slightly more descriptive message?

Reply:
Yes, I'll fix that. Debugging remnant :D